/alpha_rise/waypoint_admin:
  ros__parameters:
    update_waypoint_topic: "/alpha_rise/mvp_helm/bhv_path_following/update_waypoints"
    odom_topic: "/alpha_rise/odometry/filtered"
    get_state_service: "/alpha_rise/mvp_helm/get_state"
    change_state_service: "/alpha_rise/mvp_helm/change_state"
    get_waypoint_service: "/alpha_rise/bhv_path_following/get_next_waypoints"
    check_state_update_rate: 1 #sec
    search_mode_timer: 1200 #sec
    follow_mode_timer: 1500 #sec
    fls_beamwidth: 35 #degrees 
    search_mode_initial_radius: 10.0 #m
    exit_mode_distance: 30.0 #m
    reacquisition_s_param: 0.5
    path_topic: "/alpha_rise/path"
    stand_off_distance: 20.0
    n_points: 20
    operating_depth: -15.0